### DISCUSSION TITLE
VM/370 target

### DISCUSSION THREAD

**Discussion by rvjansen** [adesutherland/CREXX#322] [Q&A]
Adrian, do we have git hashes from where the VM/370 target still worked (the non-threaded version) and when it stopped working?

**Comment by RossPatterson** [comment]
> Adrian, do we have git hashes from where the VM/370 target still worked (the non-threaded version) and when it stopped working?

Right now I'm assuming f071f402d9e8076eabc5f753748ba1a42b43da9c is the last possibly-good commit, since the next one deleted the CMS build YAML.
